src/components/Image/Image.js

```diff
@@ -33,15 +33,27 @@ const LazyLoadedImage = (props) => {
     // the low quality image (props.image.thumbnail) will be hidden
   };
 
+  // This block checks whether the user specified fallback image actually works or not
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValidFallback, setIsValidFallback] = useState(false);
+  useEffect(() => {
+    fetch(props.placeholderImg).then(res => {
+      setIsValidFallback(res.status === 200);
```

Comment:
Could it be "HTTP 304 Not Modified" in some cases?

Reply:
I've changed it to check for `res.ok` instead now, which should address this issue.